test/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ set(UR_TEST_FUZZTESTS ON CACHE BOOL "Run fuzz tests if using clang and UR_DPCXX
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 set(INSTALL_GTEST OFF)
 FetchContent_MakeAvailable(googletest)
+
+set_target_properties(gtest PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Comment:
I needed these on SLES for some reason. I'll remove them and create a separate PR once CI finishes (I don't want to force push right now to restart everything).